app/src/main/res/layout/submitted_answer_item.xml

```diff
@@ -23,75 +25,92 @@
-    app:questionSplitViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; viewModel.isSplitView}"
-    app:questionSplitViewMarginBottom="@{@dimen/margin_0}"
-    app:questionSplitViewMarginEnd="@{@dimen/margin_32}"
-    app:questionSplitViewMarginStart="@{@dimen/margin_32}"
-    app:questionSplitViewMarginTop="@{@dimen/margin_24}"
-    app:questionViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; !viewModel.isSplitView}"
-    app:questionViewMarginBottom="@{@dimen/margin_0}"
-    app:questionViewMarginEnd="@{@dimen/margin_32}"
-    app:questionViewMarginStart="@{@dimen/margin_32}"
-    app:questionViewMarginTop="@{@dimen/margin_24}">
-
-    <TextView
-      android:id="@+id/submitted_answer_text_view"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:background="@drawable/submitted_answer_background"
-      android:contentDescription="@{accessibleAnswer ?? submittedAnswer}"
-      android:paddingStart="12dp"
-      android:paddingTop="@{viewModel.hasConversationView ? @dimen/padding_16 : @dimen/padding_8}"
-      android:paddingEnd="12dp"
-      android:paddingBottom="@{viewModel.hasConversationView ? @dimen/padding_16 : @dimen/padding_8}"
-      android:text="@{submittedAnswer}"
-      android:textColor="@color/oppiaPrimaryText"
-      android:textSize="16sp"
-      android:visibility="gone" />
-
-    <androidx.recyclerview.widget.RecyclerView
-      android:id="@+id/submitted_answer_recycler_view"
+    android:orientation="horizontal">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:background="@drawable/submitted_answer_background"
-      android:paddingStart="8dp"
-      android:paddingEnd="8dp"
-      android:visibility="gone"
-      app:itemDecorator="@{@drawable/divider}"
-      app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"
-      app:list="@{submittedListAnswer.setOfHtmlStringsList}" />
-
-    <ImageView
-      android:id="@+id/answer_tick"
-      android:layout_width="24dp"
-      android:layout_height="24dp"
-      android:layout_gravity="center_vertical"
-      android:layout_marginStart="12dp"
-      android:src="@drawable/ic_check_primary"
-      android:visibility="@{viewModel.isCorrectAnswer &amp;&amp; !viewModel.hasConversationView ? View.VISIBLE : View.GONE}"
-      app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintHorizontal_bias="0.0"
-      app:layout_constraintStart_toEndOf="@+id/submitted_answer_container"
-      app:layout_constraintTop_toTopOf="parent" />
+      android:id="@+id/container"
+      android:layout_gravity="@{viewModel.hasConversationView ? Gravity.END : Gravity.START}"
```

Comment:
@BenHenning I tried shifting ```layout_gravity``` to constraintLayout without removing LinearLayout to check if it works or not. It shows some build error -  
```
e: /home/prayutsu/StudioProjects/oppia-android/app/build/generated/source/kapt/debug/org/oppia/android/DataBinderMapperImpl.java:295: error: cannot find symbol
import org.oppia.android.databinding.SubmittedAnswerItemBindingImpl;
                                    ^
  symbol:   class SubmittedAnswerItemBindingImpl
  location: package org.oppia.android.databinding
```

Can you please help me in identifying the mistake here.

Reply:
@BenHenning I also tried to search about it and found [this](https://stackoverflow.com/questions/49772680/how-to-set-layout-gravity-with-databinding) but it didn't help much